README.md

```diff
@@ -155,22 +154,24 @@ wallet.verify(message, signature); // true
 `XRPClient` is a gateway into the XRP Ledger. `XRPClient` is initialized with a single parameter, which is the URL of the remote adapter (see [Server-Side Component][#server-side-component]).
```

Comment:
should this be updated to reflect the fact that an XRPClient is initialized with more than a single param?

Reply:
I echo @amiecorso